cds-snc/covid-alert-portal
# Add Location model, build Location forms, summary screen, save

# Summary | Résumé

Add a location model to the new "register" app.

Fields for the location model can be seen on [the "summary" page](https://www.figma.com/proto/0CvVmmVb0EhnCmChzel70P/QR-Code-Registration-Website?node-id=7%3A520&scaling=min-zoom). 
Note that contact info is part of the location.

Build out Location flow:
- Add location category
- Add location name
- Add location address
- Add location contact info 
- Summary page for location and for a registrant

The Summary screen allows you to jump back and edit previously entered info. Upon submission at the Summary screen, the data will be saved to the Location table, and you will be directed to the Confirmation screen.

## How to test

- If you build it locally, make sure the QR_CODES waffle switch is active.
- Go to url "/register"
- Click start
- Fill out Registrant info (2 steps)
- Fill out Location info (4 steps)
- Summary
- Confirmed!

## Not implemented
- Email verification
- Back button
- Post code lookup
